### ISSUE TITLE
Adding a new profile: Error: unknown profile name

### ISSUE THREAD

**Issue by BarbUk** [guumaster/hostctl#57]
**Describe the bug**

Using the last version (binary or compiled), I can no longer add a new profile.

**To Reproduce**

With no profile named `test`

```
$ sudo hostctl add domains test test.com
Error: unknown profile name
```

**Expected behavior**

Domain test.com is added in a newly created `test` profile

**System (please complete the following information):**
 - OS: Linux
 - Version: 
```
$ hostctl --version
hostctl version 1.0.5
```

**Comment by guumaster** [comment]
I'll look into it, it fails with `add domains` and new profiles, if profile exists it works.

**Comment by guumaster** [comment]
fixed on `v1.0.6`, not actually a bug, but added anyway.
